sdcm/nemesis.py

```diff
@@ -1370,16 +1370,23 @@ def disrupt_modify_table(self):
 
     def disrupt_mgmt_backup(self):
         self._set_current_disruption('ManagementBackup')
-        # TODO: When cloud backup is supported - add this to the belowq 'if' statement:
+        # TODO: When cloud backup is supported - add this to the below 'if' statement:
         #  and not self.cluster.params.get('use_cloud_manager', default=None)
         if not self.cluster.params.get('use_mgmt', default=None):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
         if not self.cluster.params.get('backup_bucket_location'):
             raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
-        bucket_location_name = self.cluster.params.get('backup_bucket_location').split()
-        mgr_task = mgr_cluster.create_backup_task(location_list=['s3:{}'.format(bucket_location_name[0])])
+        cluster_backend = self.cluster.params.get('cluster_backend')
+        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
+        if cluster_backend == 'aws':
+            bucket_name = f"s3:{backup_bucket_location.split()[0]}"
```

Comment:
why configuring a parameter value that has to be splited later? why not having the simple wanted value?

Reply:
Cause it has different structure in AWS, in AWS we have a parameter per region, in GCE we use only one region and don't have this parameters per region structure

Reply:
it was introduced this way as in AWS we run (all sanity tests) with multi DC, so we need to pick one bucket, not the whole list, as it has to change the command to use multiple buckets